test/e2e/multihoming_utils.go

```diff
@@ -283,6 +283,38 @@ func multiNetIngressLimitingPolicy(policyFor string, appliesFor metav1.LabelSele
 	}
 }
 
+func multiNetIngressAllowAllPolicy(policyFor string) *mnpapi.MultiNetworkPolicy {
```

Comment:
lets wait for https://github.com/ovn-org/ovn-kubernetes/pull/4156#discussion_r1495550950
it can be reused in this PR it seems